### PR TITLE
KRAK-460: made changes that allow us to create roles for instances.

### PR DESCRIPTION
**Pull request by alejandroEsc**
WORK IN PROGRESS

### CODE REVIEW

**Comment by alejandroEsc** [comment]
please retest this

**Comment by alejandroEsc** [comment]
retest this please

**Comment by alejandroEsc** [comment]
retest this please

File: terraform/aws/aws.tf [hunk 1]
```diff
@@ -7,6 +7,131 @@ provider "aws" {
...
+    {
+      "Effect": "Allow",
+      "Action": "s3:*",
+      "Resource": [
+        "arn:aws:s3:::kubernetes-*"
+      ]
+    }
+  ]
+} 
+EOF
```
Comment (davidewatson) [inline terraform/aws/aws.tf]: ?

Comment (alejandroEsc) [inline terraform/aws/aws.tf]: yes, a placeholder for a json entry, above youll see something like 

```"some_key" <<EOF```

**Comment by spiffxp** [comment]
retest this please

**Comment by spiffxp** [comment]
lgtm, github is warning that this is out of base with `master` since we tried a rebase/squash merge with #363, but I got a clean `--no-ff merge` locally, and PR builds are happy

**Comment by alejandroEsc** [comment]
Holding off until we agree i figure out terraform role naming issue.

**Comment by spiffxp** [comment]
agreed, pr builds were passing before while cluster teardown was failing; they should now fail if teardown is still failing

**Comment by alejandroEsc** [comment]
closing since no new features to be added to kraken.

**Pull request closed without merging**